### ISSUE TITLE
[Bug Report] Datatables ignores transparent class

### ISSUE THREAD

**Issue by fnoop** [vuetifyjs/vuetify#3730]
### Versions and Environment
**Vuetify:** 1.0.11
**Vue:** 2.5.16
**Browsers:** Chrome 65.0.3325.181
**OS:** Linux

### Steps to reproduce
<v-data-table class="transparent">

### Expected Behavior
Table should have transparent background

### Actual Behavior
Table does not have transparent background, has a grey background (in dark theme)

### Reproduction Link
[https://codepen.io/anon/pen/xWjQyY](https://codepen.io/anon/pen/xWjQyY)

### Other comments
In vuetify/src/components/VDataTable/VDataTable.js, adding 'transparent' to 'datatable table' class definition works, but it should honour any other classes requested in the component call.

**Comment by fnoop** [comment]
A (ugly!) workaround:
```
  mounted () {
    const tables = document.querySelectorAll('.datatable.table, .datatable__actions')
    Object.keys(tables).forEach(key => { tables[key].className += ' transparent' })
  }
```

**Comment by jacekkarczmarczyk** [comment]
less ugly
https://codepen.io/anon/pen/pKrNrN?editors=1100

**Comment by exyzzy** [comment]
I had to use this with light theme, is there a better way?
```
.theme--light.v-table {
    background-color: transparent;
}
```

**Comment by nekosaur** [comment]
Fixed in 2.0
